docs/resources/slint-docs-preview.html

```diff
@@ -36,7 +36,9 @@
 
     async function run() {
         await slint.default();
-        var elements = document.querySelectorAll("code.language-slint, .rustdoc pre.language-slint, div.highlight-slint");
+        let selector = ["code.language-slint", ".rustdoc pre.language-slint", "div.highlight-slint"]
+            .map((sel) => `${sel}:not([class*=slint-ignore]):not([class*=slint-no_run])`).join(",");
```

Comment:
If we didn't do the class replacement, this would then be easier

Reply:
Sure, I can revert and try to do it without the class replacement. Do you prefer that?

It means registering `slint`, `slint,no_run` and `slint,ignore` as languages with highlight js and different selectors in the highlight part, I think.

Reply:
yeah

Reply:
> is that not what the patch was doing originally anyway?

Yes. But since we discussed doing the class replacement I removed that double registration again, since it wasn't necessary.

Reply:
Is that a "yes" to revert the changes, not do class replacement, btw?

Reply:
I think you said to do class replacement because :

> OTOH the split makes more sense as a one-time transformation and then we don't have to teach hljs about the slint-ignore and slint-no-run languages :slightly_smiling_face:. So I'll go with your code.

But since it actually seems to make things more complicated, it is probably better not having it.